addons/mediarecorder/userscript.js

```diff
@@ -63,6 +63,24 @@ export default async ({ addon, console, msg }) => {
       recordOptionSeconds.appendChild(recordOptionSecondsInput);
       recordOptionInner.appendChild(recordOptionSeconds);
 
+      // Delay
+      const recordOptionDelay = document.createElement("p");
+      const recordOptionDelayInput = Object.assign(document.createElement("input"), {
+        type: "number",
+        min: 1,
```

Comment:
Min should be 0

Reply:
It would also be better to make this input smaller than duration, since it's less important

Reply:
Fixed min

I feel like making the input a different size would make it look off